### PR TITLE
add delay props

### PR DESCRIPTION
**Pull request by glekner**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -45,6 +45,8 @@ class EllipisWithTooltip extends React.Component {
       <OverlayTrigger
         placement = {placement}
         overlay={tooltip}
+        delayShow={delayShow}
+        delayHide={delayHide}
```
Comment (amirfefer) [inline src/index.js]: would you mind to add default value for these props?

**Comment by glekner** [comment]
done @amirfefer

File: src/index.js [hunk 2]
```diff
@@ -57,9 +65,16 @@ class EllipisWithTooltip extends React.Component {
...
-    style: PropTypes.object,
+  placement: PropTypes.string,
+  children: PropTypes.node.isRequired,
+  style: PropTypes.object,
+  delayShow: PropTypes.number,
+  delayHide: PropTypes.number,
+};
+
+EllipisWithTooltip.defaultProps = {
+  delayHide: 0,
```
Comment (amirfefer) [inline src/index.js]: giving an undefined value wouldn't be better? don't undefined props  stop from spreading?

**Comment by amirfefer** [comment]
Thanks @glekner  !

**Pull request merged**